config/strings.yaml

```diff
@@ -81,8 +80,10 @@ error:
     title: Server Error
     heading: Uh oh, that's a 500.
     message: Perhaps try to search for something else instead?
+    sizeLimitNote:
+      message: "The document is too large to display in Library. We are working on a fix, but in the meantime, you can view the document directly in Google Drive."
```

Comment:
should we say "the document may be..." instead of "the document is..." in case all of doctools is down somehow?